### PR TITLE
Look for -XX:[+/-]UseZlibNX on AIX

### PR DESCRIPTION
**Pull request by pshipton**
AIX puts zlibNX on the LIBPATH when running on P9 or later, but having this in the environment may cause some things, such as git clone, to fail when exec'ed from Java. For example,
`fatal: pack has bad object at offset 3872100: inflate returned -5`

Backport of https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/664

### CODE REVIEW

File: jdk/src/solaris/bin/java_md_solinux.c [hunk 1]
```diff
@@ -634,7 +657,7 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                     /* On AIX we additionally need 'jli' in the path because ld doesn't support $ORIGIN. */
                     JLI_StrLen(jrepath) + JLI_StrLen(arch) + JLI_StrLen("/lib//jli:") +
                     /* On AIX P9 or newer with NX accelerator enabled, add the accelerated zlibNX to LIBPATH */
```
Comment (keithc-ca) [inline jdk/src/solaris/bin/java_md_solinux.c:659]: While you're here, can you finish the sentence with a period?

Comment (pshipton) [reply jdk/src/solaris/bin/java_md_solinux.c]: Updated for all versions and created https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/669

**Comment by keithc-ca** [comment]
Jenkins compile aix jdk8

**Comment by keithc-ca** [comment]
I guess I was just a little impatient: It's done now.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**